backend/src/test/java/com/morak/back/auth/application/FakeOAuthClient.java

```diff
@@ -0,0 +1,33 @@
+package com.morak.back.auth.application;
+
+import com.morak.back.auth.application.dto.OAuthAccessTokenResponse;
+import com.morak.back.auth.application.dto.OAuthMemberInfoResponse;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpClientErrorException;
+
+@Component
+@Primary
```

Comment:
넵 맞습니다 ~

```
Indicates that a bean should be given preference when multiple candidates 
are qualified to autowire a single-valueddependency. If exactly one 'primary' bean
exists among the candidates, it will be the autowired value.
```